pkg/web/templates/components/transaction_rejection.html

```diff
@@ -11,7 +11,8 @@ <h3 class="font-bold text-xl">Reject Transaction</h3>
       <form method="dialog" hx-post="">
         <div class="my-4">
           <label for="transaction-rejection" class="label-style">Select a reason for rejecting the transaction:</label>
-          <select id="transaction-rejection" name="code" class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5">
+          <select id="transaction-rejection" name="code"
```

Comment:
Formatting change.

Reply:
The down carrot on the select box seems to have no padding applied to it and is too close to the right border of the select; is it possible to give it the same padding as the label on the left?